Build fail looks real: 
```
======================================================================

ERROR: test suite for <class 'casexml.apps.phone.tests.test_sync_logs.SyncLogModelTest'>

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 209, in run

    self.setUp()

  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 292, in setUp

    self.setupContext(ancestor)

  File "/vendor/lib/python2.7/site-packages/nose/suite.py", line 315, in setupContext

    try_run(context, names)

  File "/vendor/lib/python2.7/site-packages/nose/util.py", line 471, in try_run

    return func()

  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_logs.py", line 120, in setUpClass

    cls.restore_user = create_restore_user(cls.domain)

  File "corehq/ex-submodules/casexml/apps/phone/tests/utils.py", line 35, in create_restore_user

    'something': 'arbitrary'

  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1459, in create

    commcare_user.save(**get_safe_write_kwargs())

  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1404, in save

    super(CommCareUser, self).save(**params)

  File "/mnt/commcare-hq-ro/corehq/apps/users/models.py", line 1263, in save

    raise self.Inconsistent("CouchUser with username %s already exists" % self.username)

Inconsistent: CouchUser with username mclovin already exists
```

--- a/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_logs.py
+++ b/corehq/ex-submodules/casexml/apps/phone/tests/test_sync_logs.py
@@ -1,3 +1,4 @@
+import uuid
 from django.test import TestCase, SimpleTestCase
 from casexml.apps.case.xml import V1, V2
 from casexml.apps.phone.models import SyncLog, CaseState
@@ -117,7 +118,7 @@ class SyncLogModelTest(TestCase):
     def setUpClass(cls):
         cls.project = Domain(name=cls.domain)
         cls.project.save()
-        cls.restore_user = create_restore_user(cls.domain)
+        cls.restore_user = create_restore_user(cls.domain, username=uuid.uuid4().hex)
 
     @classmethod
     def tearDownClass(cls):

